ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Protocol/ChainSel.hs

```diff
@@ -76,9 +69,9 @@ acrossEras ::
   -> Ordering
 acrossEras (WithBlockNo bnoL (WrapSelectView l))
            (WithBlockNo bnoR (WrapSelectView r)) = \case
-    CompareBlockNo     -> compare bnoL bnoR
-    CustomChainSel f   -> f l r
```

Comment:
I would guess `CustomChainSel` doesn't occur in the codebase now.

Should we remove it, and only add it back if/when the future day arises where we actually need it?

Reply:
Sounds good, adding back `CustomChainSel` will probably the easiest part when we have pairs of blocks/eras that can neither use `CompareBlockNo` nor `CompareSameSelectView`.